src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -316,6 +317,7 @@ const Inventory = ({
 
       groups = {
         ...groups[0],
+        title: isWorkSpaceEnabled && 'Workspaces',
```

Comment:
```suggestion
        title: isWorkSpaceEnabled ? 'Workspaces' : 'Groups',
```

Reply:
Otherwise, when the flag is disabled groups column would have an empty header title